src/engine/plugins/tile.cpp

```diff
@@ -865,6 +934,66 @@ void encodeVectorTile(const DataFacadeBase &facade,
                 }
             }
         }
+
+        {
+            protozero::pbf_writer line_layer_writer(tile_writer, util::vector_tile::LAYER_TAG);
```

Comment:
wouldn't it be possible to simply add a tag to the already present lines? We add many attributes to lines, why not segregated as well?

Reply:
Also: I am a bit confused about why our travis tests fail for systems in a more or less random distribution based on tile size.

Reply:
I'm also confused about randomly failed Monaco tile size tests. Will try to investigate it.

Reply:
Actually, I can add segregated flag to the present lines (sections). I thought that separate drawing layer will be a better logic choice rather than modifying current structure. What is your opinion?

Reply:
And the separate segregated layer is easier to make optional when retrieving tiles.